<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Instal·lador</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../usr/share/kademar/icons/kpackage.png</normaloff>../../../../../usr/share/kademar/icons/kpackage.png</iconset>
  </property>
  <widget class="QLabel" name="fons" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/instalador-nano/img/bg/instalador-nano.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="w_process" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="l_titol" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Procés
   d'Instal·lació</string>
    </property>
   </widget>
   <widget class="QLabel" name="led_disc" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>99</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="pixmap" >
     <pixmap>../../../../../usr/share/kademar/utils/instalador-nano/img/vermell_p.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_disc" >
    <property name="geometry" >
     <rect>
      <x>53</x>
      <y>98</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text" >
     <string>Disc dur i
  Partició</string>
    </property>
   </widget>
   <widget class="QLabel" name="led_copia" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>180</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="pixmap" >
     <pixmap>../../../../../usr/share/kademar/utils/instalador-nano/img/vermell_p.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_copia" >
    <property name="geometry" >
     <rect>
      <x>53</x>
      <y>173</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text" >
     <string>Còpia de
  fitxers</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" native="1" name="w_logo" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>300</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="logo_kademar" >
    <property name="geometry" >
     <rect>
      <x>21</x>
      <y>88</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="pixmap" >
     <pixmap>../../../../../usr/share/kademar/utils/instalador-nano/img/kademar.png</pixmap>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>38</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading</set>
    </property>
    <property name="textVisible" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_finished" >
    <property name="geometry" >
     <rect>
      <x>3</x>
      <y>41</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Copia Finalitzada</string>
    </property>
    <property name="scaledContents" >
     <bool>false</bool>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" native="1" name="w_botons" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>420</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QPushButton" name="boto_next" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text" >
     <string>Següent</string>
    </property>
    <property name="icon" >
     <iconset>
      <normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/next.png</normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/next.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="default" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boto_exit" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sortir</string>
    </property>
    <property name="icon" >
     <iconset>
      <normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/exit.png</normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/exit.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="pages" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>0</y>
     <width>491</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>1</number>
   </property>
   <widget class="QWidget" name="p_principal" >
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>60</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text" >
      <string>Bienvenidos al Instalador de kademar en USB</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>300</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text" >
      <string>Este asistente te ayudará a instalar kademar Linux
 en un dispositivo USB.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>160</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>../../../../../usr/share/kademar/utils/instalador/img/lyra.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_18" >
     <property name="geometry" >
      <rect>
       <x>211</x>
       <y>167</y>
       <width>52</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>../../../../../usr/share/kademar/utils/instalador/img/arrow.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_19" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>150</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>../../../../../usr/share/kademar/utils/instalador-nano/img/usbpendrive_unmount.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="p_disc" >
    <widget class="QLabel" name="l_titol_2" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>35</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Seleccione la partición USB donde instalar kademar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boto_particionar" >
     <property name="geometry" >
      <rect>
       <x>290</x>
       <y>110</y>
       <width>111</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text" >
      <string>Particionar</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/gparted.png</normaloff>../../../../../usr/share/kademar/utils/instalador-nano/img/gparted.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_hd_arrel" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>200</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>170</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Partición del dispositivo USB</string>
     </property>
    </widget>
    <widget class="QFrame" name="f_final_2" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>270</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="cb_format" >
      <property name="geometry" >
       <rect>
        <x>33</x>
        <y>26</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Formatear la particion</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="p_mbr" >
    <widget class="QLabel" name="label_30" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>80</y>
       <width>381</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text" >
      <string>Actualmente se están copiando los ficheros en el USB.

Puede ver el progreso, mediante la barra de progreso situada en el lado izquierdo.


Una vez se haya copiado se instalará el gestor de arranque.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_titol_6" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>35</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Copiando</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="p_final" >
    <widget class="QLabel" name="l_titol_5" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>35</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Instalación FInalizada</string>
     </property>
    </widget>
    <widget class="QFrame" name="f_final" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>250</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="cb_reboot" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>26</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Reiniciar l'ordinador</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_32" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>60</y>
       <width>441</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text" >
      <string>kademar ya está instalada en el dispositivo USB.

Si quiere puede apagar el instalador y continuar usando kademar.
O si quiere, puede reiniciar el ordenador.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>boto_next</tabstop>
  <tabstop>boto_exit</tabstop>
  <tabstop>boto_particionar</tabstop>
  <tabstop>cb_hd_arrel</tabstop>
  <tabstop>cb_reboot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
